sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_create_test_live.cpp

```diff
@@ -262,7 +264,7 @@ TEST_F(KeyVaultKeyClient, CreateRsaHsmKey)
   // This client requires an HSM client
   CreateHsmClient();
   auto const& client = GetClientForTest(keyName);
-
+  
```

Comment:
Newline is good, but there seem to be 2 trailing spaces here, did clang-format allow this?